qa/tasks/cephfs/test_layout.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+class TestInheritedLayout(CephFSTestCase):
+
+    def test_layout_root(self):
+        self.mount_a.run_shell(["mkdir", "a"])
+        layout = self.mount_a.getfattr("a", "ceph.dir.layout") 
```

Comment:
What are we testing exactly here?

Reply:
Test if dir a inherits "/" directories layout.

Reply:
but nothing is actually verified by the test?